### ISSUE TITLE
Please help

### ISSUE THREAD

**Issue by aliok222** [kohya-ss/musubi-tuner#241]
Hi this is a week I'm trying train a lora style but i do not know if my setup is correct even with GPT i could not figure it out , any help appreciated 
Im not sure do i need use video as data set and caption or frame this is a man transform to werewolf , Thank you  

**musubi-tuner**
my datasets 120 files and caption 
"C:\ai\musubi-tuner\datasets\my_lora_dataset\frame_00002.png"
"C:\ai\musubi-tuner\datasets\my_lora_dataset\frame_00002.txt"
"C:\ai\musubi-tuner\datasets\my_lora_dataset\frame_00001.png"
"C:\ai\musubi-tuner\datasets\my_lora_dataset\frame_00001.txt"

Models i use 
"C:\ai\musubi-tuner\models\wan_2.1_vae.safetensors"
"C:\ai\musubi-tuner\models\wan2.1_i2v_720p_14B_bf16.safetensors"
"C:\ai\musubi-tuner\models\models_clip_open-clip-xlm-roberta-large-vit-huge-14.pth"
"C:\ai\musubi-tuner\models\models_t5_umt5-xxl-enc-bf16.pth"

**Training Comfig Toml** 
[general]
output_dir = "C:/ai/musubi-tuner/outputs/my_werewolf_lora"
logging_dir = "C:/ai/musubi-tuner/logs"
mixed_precision = "bf16"
seed = 42

[model]
pretrained_model_path = "C:/ai/musubi-tuner/models/wan2.1_i2v_720p_14B_bf16.safetensors"
vae = "C:/ai/musubi-tuner/models/wan_2.1_vae.safetensors"
t5 = "C:/ai/musubi-tuner/models/models_t5_umt5-xxl-enc-bf16.pth"
clip = "C:/ai/musubi-tuner/models/models_clip_open-clip-xlm-roberta-large-vit-huge-14.pth"
dit = "C:/ai/musubi-tuner/models/wan2.1_i2v_720p_14B_bf16.safetensors"
task = "i2v-14B"

[data]
train_data_dir = "C:/ai/musubi-tuner/datasets/my_lora_dataset"
caption_extension = ".txt"
resolution = [512, 512]

[training]
network_dim = 128
network_alpha = 64
learning_rate = 0.0001
train_batch_size = 4
max_train_steps = 2000
save_every_n_steps = 500
output_name = "werewolf_lora"

**Data sets comfig Toml** 
train_data_dir = "C:/ai/musubi-tuner/datasets/my_lora_dataset"
resolution = [512, 512]
caption_extension = ".txt"
num_repeats = 1
shuffle_caption = true
